src/Cli/Microsoft.DotNet.Cli.Utils/RuntimeEnvironment.cs

```diff
@@ -13,7 +13,9 @@ internal enum Platform
         Windows = 1,
         Linux = 2,
         Darwin = 3,
-        FreeBSD = 4
+        FreeBSD = 4,
+        illumos = 5,
```

Comment:
Is this supposed to start with a lower case character?

Reply:
The official casing is lowercase (to disambiguate letters `i` and lowercase `l`, like iOS). In the output we print the value as is.